### ISSUE TITLE
Issue with slf4j and version 4.25.0

### ISSUE THREAD

**Issue by nikola-miletic** [liquibase/liquibase-hanadb#175]
After upgrade to 4.25.0 I am getting error:
`Caused by: java.lang.IllegalArgumentException: LoggerFactory is not a Logback LoggerContext but Logback is on the classpath. Either remove Logback or the competing implementation (class org.slf4j.simple.SimpleLoggerFactory loaded from file:~/gradle/caches/modules-2/files-2.1/org.liquibase.ext/liquibase-hanadb/4.25.0/4bbb64f5e24a5312d0cb8dcbe9d92d1f87d805be/liquibase-hanadb-4.25.0.jar)`

I am using logback in project and logs looks like:

```
SLF4J: Class path contains multiple SLF4J providers.
SLF4J: Found provider [ch.qos.logback.classic.spi.LogbackServiceProvider@79d8407f]
SLF4J: Found provider [org.slf4j.simple.SimpleServiceProvider@5fbe4146]
SLF4J: See https://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual provider is of type [org.slf4j.simple.SimpleServiceProvider@5fbe4146]
```

When I checked the jar file I was able to see this:
<img width="741" alt="Screenshot 2023-11-27 at 18 35 01" src="https://github.com/liquibase/liquibase-hanadb/assets/43137517/93d3e5b5-176b-4fa8-ad8d-16d0620b91ea">

Is this a bug?

**Comment by stephanme** [comment]
Same for me. 

Issue might be caused by referencing the [liquibase-parent-pom](https://github.com/liquibase/liquibase-parent-pom/blob/main/pom.xml)

**Comment by nikola-miletic** [comment]
@stephanme looks like this is fixed in 4.25.1
https://github.com/liquibase/liquibase-hanadb/releases/tag/untagged-b760611478607cbf24ca

**Comment by stephanme** [comment]
Yes 4.25.1 works for me as well again. Thanks for fixing.
